### DISCUSSION TITLE
non-latin fonts invisible in a database column

### DISCUSSION THREAD

**Discussion by solsem** [sqlitebrowser/sqlitebrowser#2819] [Q&A]
Hi folks,
I have an sql database which is a dictionary.
In my desktop and laptop computers I have installed the same version of sqlite browser and both computers have the same fonts and the same version of operative system Windows 10.
In the desktop the column that has  non-english fonts(Ethiopic Unicode) shows the data fine but not in the laptop. Also when I try to write in the column that has Ethiopic unicode fonts, I see the cursor moves right but see no font written, not even empty boxes!
Why would two computers, a desktop and laptop that have the same versions of windows 10 and DB browser with the same fonts 
show different results? The database could not be corrupted as I am using the same database.
In Notepad++, I can write using Ethiopic unicode fonts in both computers with out problem. It is only in SQLlite browser of the laptop
the Ethiopic unicode fonts not showing as you can see in the attached files.

![notWorking](https://user-images.githubusercontent.com/89905972/131647177-4ccff729-c979-4582-8a9d-8eabbe9e79d3.PNG)
![working](https://user-images.githubusercontent.com/89905972/131647178-98f19ca4-3a49-4034-8946-30e313cff199.JPG)

**Comment by chrisjlocke** [comment]
This might be the font used to display the text.
In Preferences (Click 'Edit' then 'Preferences') then in the 'Data Browser' tab, what font have you got selected?

![image](https://user-images.githubusercontent.com/7872218/131657341-291a0d40-67fc-4966-9c92-c0aadf421e9b.png)

I'm not sure what font might be better, but try different truetype ones, and hopefully the missing letters should return.

**Comment by solsem** [comment]
Hi Chris, I was using the default MS Shell Dlg 2. But I did as you said and chose Abyssinica SIL and voilaIt works!Thank you very much! Cheers! Skickades från

**Reply by chrisjlocke** [reply]
Yay!  Glad it worked!  👍
